doc-files/release-notes-18.md

```diff
@@ -0,0 +1,110 @@
+[JDK-8276915](https://bugs.openjdk.java.net/browse/JDK-8276915)|Crash on iOS 15.1 in GlassRunnable::dealloc|graphics
+[JDK-8278905](https://bugs.openjdk.java.net/browse/JDK-8278905)|JavaFX: EnumConverter has a typo in the toString method|graphics
+[JDK-8279328](https://bugs.openjdk.java.net/browse/JDK-8279328)|CssParser uses default charset instead of UTF-8|graphics
+[JDK-8253351](https://bugs.openjdk.java.net/browse/JDK-8253351)|MediaPlayer does not display an mp4 if there no speakers connected to the PC's|media
+[JDK-8268718](https://bugs.openjdk.java.net/browse/JDK-8268718)|[macos] Video stops, but audio continues to play when stopTime is reached|media
+[JDK-8222455](https://bugs.openjdk.java.net/browse/JDK-8222455)|JavaFX error loading glass.dll from cache|other
+[JDK-8270839](https://bugs.openjdk.java.net/browse/JDK-8270839)|Remove deprecated implementation methods from Scene|scenegraph
+[JDK-8268849](https://bugs.openjdk.java.net/browse/JDK-8268849)|Update to 612.1 version of WebKit|web
+[JDK-8270479](https://bugs.openjdk.java.net/browse/JDK-8270479)|WebKit 612.1 build fails with Visual Studio 2017|web
+[JDK-8272329](https://bugs.openjdk.java.net/browse/JDK-8272329)|Cherry pick GTK WebKit 2.32.3 changes|web
+[JDK-8274107](https://bugs.openjdk.java.net/browse/JDK-8274107)|Cherry pick GTK WebKit 2.32.4 changes|web
+[JDK-8275138](https://bugs.openjdk.java.net/browse/JDK-8275138)|WebView: UserAgent string is empty for first request|web
+[JDK-8276847](https://bugs.openjdk.java.net/browse/JDK-8276847)|JSException: ReferenceError: Can't find variable: IntersectionObserver|web
+[JDK-8277133](https://bugs.openjdk.java.net/browse/JDK-8277133)|Dragboard contents retrieved all over again during a DND process on WebView|web
+[JDK-8277457](https://bugs.openjdk.java.net/browse/JDK-8277457)|AccessControlException: access denied ("java.net.NetPermission" "getCookieHandler")|web
+[JDK-8160597](https://bugs.openjdk.java.net/browse/JDK-8160597)|IllegalArgumentException when we initiate drag on Image|window-toolkit
+[JDK-8227371](https://bugs.openjdk.java.net/browse/JDK-8227371)|Drag&Drop while holding the CMD key does not work on macOS|window-toolkit
+[JDK-8242544](https://bugs.openjdk.java.net/browse/JDK-8242544)|CMD+ENTER key event crashes the application when invoked on dialog|window-toolkit
+[JDK-8269967](https://bugs.openjdk.java.net/browse/JDK-8269967)|JavaFX should fail fast on macOS below minimum version|window-toolkit
+[JDK-8269968](https://bugs.openjdk.java.net/browse/JDK-8269968)|[REDO] Bump minimum version of macOS for x64 to 10.12|window-toolkit
+[JDK-8271398](https://bugs.openjdk.java.net/browse/JDK-8271398)|GTK3 drag view image swaps red and blue color channels|window-toolkit
+[JDK-8274929](https://bugs.openjdk.java.net/browse/JDK-8274929)|Crash while reading specific clipboard content|window-toolkit
+[JDK-8275723](https://bugs.openjdk.java.net/browse/JDK-8275723)|Crash on macOS 12 in GlassRunnable::dealloc|window-toolkit
+
```

Comment:
[REDO] Wrong stage gets focused after modal stage creation https://bugs.openjdk.java.net/browse/JDK-8271054 ?

Reply:
No, that fix is in 19. These release notes are for 18.

Reply:
Sorry, my mistake.